doc/deploy_starrocks_with_operator_howto.md

```diff
@@ -0,0 +1,297 @@
+| Kind      | The resource type of the object. The value must be `StarRocksCluster`.                                                                                                                                                                             |
+| Metadata  | Metadata, in which the following sub-fields are nested:<ul><li>`name`: the name of the object. Each object name uniquely identifies an object of the same resource type.</li><li>`namespace`: the namespace to which the object belongs.</li></ul> |
+| Spec      | The expected status of the object. Valid values are `starRocksFeSpec`, `starRocksBeSpec`, and `starRocksCnSpec`.                                                                                                                                   |
+
+You can also deploy the StarRocks cluster by using a modified configuration file. For supported fields and detailed
+descriptions, see [api.md](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/api.md).
+
+Deploying the StarRocks cluster takes a while. During this period, you can use the
+command `kubectl -n starrocks get pods` to check the starting status of the StarRocks cluster. If all the pods are in
+the `Running` state and all containers inside the pods are `READY`, the StarRocks cluster is running as expected.
+
+> **NOTE**
+>
+> If you customize the namespace in which the StarRocks cluster is located, you need to replace `starrocks` with the
+> name of your customized namespace.
+
+```bash
+$ kubectl -n starrocks get pods
+NAME                                  READY   STATUS    RESTARTS   AGE
+starrocks-controller-65bb8679-jkbtg   1/1     Running   0          22h
+starrockscluster-sample-be-0          1/1     Running   0          23h
+starrockscluster-sample-be-1          1/1     Running   0          23h
+starrockscluster-sample-be-2          1/1     Running   0          22h
+starrockscluster-sample-fe-0          1/1     Running   0          21h
+starrockscluster-sample-fe-1          1/1     Running   0          21h
+starrockscluster-sample-fe-2          1/1     Running   0          22h
+```
+
+> **Note**
+>
+> If some pods cannot start after a long period of time, you can use `kubectl logs -n starrocks <pod_name>` to view the
+> log information or use `kubectl -n starrocks describe pod <pod_name>` to view the event information to locate the
+> problem.
+
+## 3. Manage StarRocks Cluster
+
+### 3.1. Access StarRocks Cluster
+
+The components of the StarRocks cluster can be accessed through their associated Services, such as the FE Service. For
+detailed descriptions of Services and their access addresses,
+see [api.md](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/api.md)
+and [Services](https://kubernetes.io/docs/concepts/services-networking/service/).
+
```

Comment:
add a bit more explanation on these two services. user should only use the `starrockscluster-sample-fe-service` for external usage.